test/unit/videoSource.test.js

```diff
@@ -430,7 +424,7 @@ describe('test isCodecAlreadyExist method', () => {
       });
 
       const srcs = source.generateSources();
-      expect(srcs[0].src).toEqual('http://res.cloudinary.com/demo/video/upload/vc_vp9/sea_turtle.webm');
+      expect(srcs[0].src).toEqual('http://res.cloudinary.com/demo/video/upload/w_400/vc_vp9/sea_turtle.webm');
```

Comment:
Again, looks like a bug.A transformation as provided but have not been added to the url